style.css

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap');
+
+:root {
+  --white: #ffffff;
+  --light-gray: #f1f2f6;
+  --orange: #ff8600;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+  background-color: var(--light-gray);
```

Comment:
Ovo je malo problematicno. Zasto bi svi elementi imali isti bg color i font-size? Vjerojatno ces poslije to prebrisat s nekim drugim propertyjima